root/static/scripts/common/entity.js

```diff
@@ -45,7 +45,8 @@ import formatTrackLength from './utility/formatTrackLength';
         }
 
         toJSON() {
-            var key, result = {};
+            let key;
+            const result = {};
             for (key in this) {
```

Comment:
you can remove `let key;` above and do
```suggestion
             for (const key in this) {
```